hbase-server/src/test/java/org/apache/hadoop/hbase/master/assignment/TestDeadServerMetricRegionChore.java

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
nit: /*

Reply:
You mean change "/**" to "/*" ?    I see other tests all use "/**" no one use "/*"

Reply:
_/**_ is interpreted as a javadoc in IDEs. If you notice others' changes, people are switching to _/*_ incrementally in their patches (files they are touching). Now that you are adding a new class, it is better to start with _/*_.

Reply:
OK, thanks for you explanation.